src/components/autocomplete/js/autocompleteController.js

```diff
@@ -203,7 +207,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       $mdUtil.enableScrolling();
     }
 
-    angular.element($window).off('resize', positionDropdown);
+    angular.element($window).off('resize', onWindowResize);
```

Comment:
It could be, but this can also damage the smoothness (we have right) now.

If so, then we should also do this change in the tabs
> https://github.com/angular/material/blob/f7d6d1007f608005c83fc22a4f59fd4868aca2dd/src/components/tabs/js/tabsController.js#L109